SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DynamicSubobjectsTest/ReplicatedGASTestActor.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h"
+
+#include "ReplicatedGASTestActor.generated.h"
+
+UCLASS()
+class AReplicatedGASTestActor : public AReplicatedTestActorBase
```

Comment:
How come this is called a GASTestActor when it has no GAS enabled / abilities granted?

Reply:
I added the AbilitySystem and the TestAtributtes in blueprints as I could not access them here.